godel_msgs/msg/BlendingPlanParameters.msg

```diff
@@ -1,12 +1,16 @@
 # Path Parameters
-float64 tool_radius             # (m) actual radius of blending tool
+float64 tool_radius           # (m) actual diameter of blending tool
```

Comment:
I have no idea why this was changed, everything in the code suggests radius means radius. Will fix.

+1